shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt

```diff
@@ -64,4 +66,10 @@ data class FeedingPointModel(
                 }
             }
         }
+
+    companion object {
+        private const val NEAREST_DISTANCE_KM = 10
+        fun FeedingPointModel.isNearTo(location: MapLocation): Boolean =
```

Comment:
Let's move this function to location module and detach it from feeding points

Reply:
Moved out to extension function at library > geolocation package